themes/kf/sass/src/index.html

```diff
@@ -179,7 +179,7 @@ <h4>Multi-framework</h4>
 				</div>
 				<div class="text">
 					<h4>Community</h4>
-					<p>A TensorFlow Serving container to export trained TensorFlow models to Kubernetes. We also integrate with Seldon Core, the open source platform for deploying machine learning models on Kubernetes.</p>
+					<p>A TensorFlow Serving container to export trained TensorFlow models to Kubernetes. We also integrate with Seldon Core, an open source platform for deploying machine learning models on Kubernetes, and NVIDIA TensorRT Inference Server for maximized GPU utilization when deploying ML/DL models at scale.</p>
```

Comment:
Sorry wasn't clear in my previous comment. This file should not need any changes. The previous file change themes/kf/layouts/index.html should be sufficient.